### PR TITLE
Document #1540

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
this pr ads a xfailing test for #1540 based on the bug report

fixing it is dependent on #1534 and followups

### CODE REVIEW

File: testing/test_mark.py [hunk 1]
```diff
@@ -563,6 +563,29 @@ def assert_markers(self, items, **expected):
                                 if isinstance(v, MarkInfo)])
             assert marker_names == set(expected_markers)
 
+    @pytest.mark.xfail(reason='callspec2.setmulti missuses keywords')
```
Comment (The-Compiler) [inline testing/test_mark.py]: nitpick: missuses -> misuses

(TIL ["missuses" is a colloquial form of "wives"](https://en.wiktionary.org/wiki/missus#English) too)

**Comment by nicoddemus** [comment]
Nice work

**Comment by nicoddemus** [comment]
Anything else here? Can we merge this?

**Comment by RonnyPfannschmidt** [comment]
i was jsut waiting for someone to merge it

**Comment by nicoddemus** [comment]
I guess everyone was waiting on the nitpick mentioned by @The-Compiler :grin: 

Feel free to go ahead if you don't have the time to fix it now, it is a nitpick after all

**Comment by RonnyPfannschmidt** [comment]
i#ll fix it and do the merge :)

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus wasn't the py26 issue fixed ? or do we need that master merge finalized

**Comment by nicoddemus** [comment]
[Seems like it](https://travis-ci.org/pytest-dev/pytest/builds/131231974)... do you have changes on `tox.in` in your branch perhaps?

**Comment by RonnyPfannschmidt** [comment]
nah, this is features branch vs master branch :/

**Comment by nicoddemus** [comment]
Oh wait, this PR is targeting `features`, didn't realize that. Then yes, we will need that merged first, but feel free to merge this now and get it out of your plate, we know the failure is unrelated. :grin:

**Pull request merged**